src/cmd-buildextend-installer

```diff
@@ -309,20 +318,20 @@ def generate_iso():
         # s390x's z/VM CMS files are limited to 8 char for filenames and extensions
         # Also it is nice to keep naming convetion with Fedora/RHEL for existing users and code
         shutil.move(os.path.join(tmpisoimages, kernel_img), os.path.join(tmpisoimages, 'kernel.img'))
-        shutil.move(os.path.join(tmpisoimages, initramfs_img), os.path.join(tmpisoimages, 'initrd.img'))
+        shutil.move(initramfs, os.path.join(tmpisoimages, 'initrd.img'))
         kernel_img = 'kernel.img'
         initramfs_img = 'initrd.img'
```

Comment:
I think the comment here about the limitation on s390x is significant. If we move the below code to use `initramfs` we throw that away don't we?

Reply:
The comment is about the destination, I'm changing the variable used for the source.  I don't see the problem?

Reply:
As far as I can tell the `initramfs_img = 'initrd.img'` variable never gets used in an s390x context after this. `kernel_img` gets used in the `-i` arg to `mk-s390-cdboot`.

Reply:
AFAIK it should be fine for s390x.  It is really about the "destinations" and it should be safe from that side. CC'ing @Prashanth684 if I have missed something.

Reply:
Thanks! Do we also need to use initramfs_img again in the `os.stat()` call?

```
# generate .addrsize file for LPAR                                         
with open(os.path.join(tmpisoimages, 'initrd.addrsize'), 'wb') as addrsize:
    addrsize_data = struct.pack(">iiii", 0, int(INITRD_ADDRESS, 16), 0,    
                                os.stat(initramfs).st_size)                
    addrsize.write(addrsize_data)                                          
```

If you've looked at it and the answer is "no" then I'm 👍

Reply:
Oh I think you're right; took another swing at reworking this.  (It'd probably be cleaner to just name them that way in the first place at the top, but trying not to scope creep this)